### PR TITLE
triggers: update WG meeting time

### PR DESCRIPTION
**Pull request by dibyom**
We decided to move the WG time by an hour (and move it to Wednesdays) to make it a bit more convenient for all the regular attendees!

/cc @khrm @savitaashture @wlynch

### CODE REVIEW

File: working-groups.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ This is the [`tektoncd/triggers`](https://github.com/tektoncd/triggers) working
...
 | Community Meeting VC       | [https://meet.google.com/oqm-imjh-uzu](https://meet.google.com/oqm-imjh-uzu)                                                                                                                                                              |
-| Community Meeting Calendar | Tuesdays every week, 12:00p-12:30p PST <br>[Calendar](https://calendar.google.com/event?action=TEMPLATE&tmeid=ZmNmbWtnNGxzcG1uaGUwNm9rMHB1MzR2ZnJfMjAyMDAxMjhUMTcwMDAwWiBnb29nbGUuY29tX2Qzb3Zjdm8xcDMyMTloOTg5NTczdjk4Zm5zQGc&tmsrc=google.com_d3ovcvo1p3219h989573v98fns%40group.calendar.google.com&scp=ALL) |
+| Community Meeting Calendar | Wednesdays every week, 11:00p-11:30p EST <br>[Calendar](https://calendar.google.com/event?action=TEMPLATE&tmeid=ZmNmbWtnNGxzcG1uaGUwNm9rMHB1MzR2ZnJfMjAyMDAxMjhUMTcwMDAwWiBnb29nbGUuY29tX2Qzb3Zjdm8xcDMyMTloOTg5NTczdjk4Zm5zQGc&tmsrc=google.com_d3ovcvo1p3219h989573v98fns%40group.calendar.google.com&scp=ALL) |
```
Comment (bobcatfish) [inline working-groups.md:96]: https://calendar.google.com/event?action=TEMPLATE&tmeid=ZDA3YzZldXZqZ2M1ajBndWVkNzRlNTdzN2ZfMjAyMDExMThUMTYwMDAwWiBnb29nbGUuY29tX2Qzb3Zjdm8xcDMyMTloOTg5NTczdjk4Zm5zQGc&tmsrc=google.com_d3ovcvo1p3219h989573v98fns%40group.calendar.google.com&scp=ALL

i think the calendar link might need an update too - i dont really understand how it works but the current link seems to want to add a repeating even on tuesdays, the one above starts this week on wednesdays

(maybe they both work? not sure!)

**Review by bobcatfish — COMMENTED** [review]
I manually updated the calendar - hopefully we'll have https://github.com/tektoncd/community/issues/38 one day :D

File: working-groups.md (hunk 1 above)
Comment (dibyom) [reply working-groups.md]: done!

**Comment by bobcatfish** [comment]
Thanks @dibyom !

/lgtm

**Pull request merged**